### PR TITLE
Implement onWebAppManifest in TabWebContentsDelegate

### PR DESCRIPTION
**Pull request by mshin-wolvic**

### CODE REVIEW

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java [hunk 1]
```diff
@@ -16,8 +19,14 @@
 import org.chromium.url.GURL;
 import org.chromium.wolvic.Tab;
 import org.chromium.wolvic.WolvicWebContentsDelegate;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
 
 public class TabWebContentsDelegate extends WolvicWebContentsDelegate {
+    private static final String TAG = "TabWebContentsDelegate";
```
Comment (svillar) [inline app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java:28]: In Wolvic we use LOGTAG instead. And we create it this way
`private static final String LOGTAG = SystemUtils.createLogtag(TabWebContentsDelegate.class);`

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java [hunk 2]
```diff
@@ -77,7 +86,20 @@ public void navigationStateChanged(int flags) {
     }
 
     @Override
-    public void onWebAppManifest(WebContents webContents, @NonNull String manifest) {}
+    public void onWebAppManifest(WebContents webContents, @NonNull String manifest) {
+         @Nullable WSession.ContentDelegate delegate = mSession.getContentDelegate();
+        if (delegate != null) {
```
Comment (svillar) [inline app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java:91]: You can do the reverse check and do an early return to save some indentation

**Review by svillar — CHANGES_REQUESTED** [review]
Nice! Some changes are needed though

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java (hunk 1 above)
Comment (mshin-wolvic) [reply app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java]: Done

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java (hunk 2 above)
Comment (mshin-wolvic) [reply app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java]: Done

**Review by svillar — APPROVED** [review]
Thanks!

**Pull request merged**